src/main/features/dashboard/views/macro/claimStatus/settlementAgreementClaimant.njk

```diff
@@ -80,6 +81,18 @@
       </form>
 {% endmacro %}
 
+{% macro settlementAgreementRejectedClaimantClaimDetails(claim, csrf) %}
+    <h2 class="heading-small">{{ t('The defendant has rejected your settlement agreement') }}</h2>
+    <p>{{ t('You can request a County Court Judgment (CCJ) against them based on the repayment plan shown in the agreement.') }}</p>
+    <p>{{ t('The court will make an order requiring them to pay the money. It does not guarantee that they pay
+      it.') }}</p>
+    <p>{{ t('{{ defendantName }} can still sign the settlement agreement until you request a CCJ.', { defendantName: claim.claimData.defendant.name }) }}</p>
+      <form novalidate method="post">
```

Comment:
doesn't need to be a form. internalLink  with button css will be consistent with other parts of the site